mlrun/utils.py

```diff
@@ -15,12 +15,12 @@
 import time
 import hashlib
 import json
-import logging
+import sys
 import pathlib
 import re
 from datetime import datetime, timezone
 from os import path, environ
-from sys import stdout
+from .clients.logger import create_logger
```

Comment:
I don't feel like new `clients` module is needed
I suggest to change `utils` to a module which will have `logger.py` file and `helpers.py` file with the rest of the stuff, everything will be imported to the module's `__init__.py` file to keep BC (clients that do `from mlrun.utils import X` won't break)
WDYT ?

Reply:
good idea, done